persistence/persistence-spark/src/main/java/org/cbioportal/persistence/spark/util/ParquetConstants.java

```diff
@@ -3,5 +3,7 @@
 public class ParquetConstants {
     // Parquet file names
     public static final String DATA_MUTATIONS = "data_mutations*.txt.parquet";
-    
+    public static final String CLINICAL_SAMPLE = "data_clinical_sample.txt.parquet";
+    public static final String CLINICAL_PATIENT = "data_clinical_patient.txt.parquet";
+    public static final String CNA_SEG = "data_cna_hg19.seg.parquet";
```

Comment:
I started working on documentation, and thought about the discussion we had last meeting about parquet file names. We can write multiple data files into one parquet file with df.write.mode("append"). I think we could document which data files would be written into the parquet file names we decide on, just like we have sql table names. For example, instead of wildcards, we could write all data_cna_*.seg files to cna_seg, data_mutations*.txt to mutations.

Reply:
SEG files we will have to leave separate based on the reference genome.  The meta file for segment files should have a reference_genome_id.

Reply:
sure we can keep them separate. Using data_cna*.seg for SEG files and also added /studies/ directory in ParquetConstants as decided in today's meeting.